### DISCUSSION TITLE
[内嵌数据库]Notion Page中内嵌Database View，博客中显示会有部分页面丢失

### DISCUSSION THREAD

**Discussion by ChenyqThu** [tangly1024/NotionNext#2018] [Q&A]
**描述bug**
1. 在Notion Next 新增一个page ，名为读书，并内嵌了书库的Database Gallery View；
2. 访问博客对应页面 [读书](https://chenge.ink/books)，发现显示的Database内容缺失了部分页面，我尝试直接以浏览器打开 /books对应的博客Page，能够正确的全部显示所有Page。
对比如下：
左侧是Blog /books页面的内容，通过浏览器直接访问开放页面获得（说明不是页面权限问题，浏览器无登陆匿名访问），右侧是博客显示的内容，对比缺失了很多的页面数据（这个书库有100+ Page，只显示了50+，大概丢了一半）
![image](https://github.com/tangly1024/NotionNext/assets/84364275/214bc4d3-36f0-453d-90bf-83886e4cd38f)

**复现步骤**
【此项必填】出现这个bug的操作步骤

**期望的正常结果**
期望能够正确显示所有页面

**截图**
【可选】相关的页面，应该的结果

**环境**

- 【必填】NotionNext版本：4.3
- 【必填】主题：Hexo
- 【必填】部署方案  VPS部署
- 【可选】操作系统:  Win/Mac/iOS/Android均同
- 【可选】浏览器：都一样

**补充说明**
【可选】与问题相关的其它说明

**Comment by ChenyqThu** [comment]
感觉问题应该和这个一样，是因为嵌入的database过大，导致默认的读取触发了1000个block的限制，所以只能显示出部分的内容。这个能指示下相关代码在哪里么，我参考#1723 修改试试看

**Comment by iSanshi** [comment]
database显示到网页上好像必须要有一个输入格式，在文档里有写

**Reply by ChenyqThu** [reply]
和格式没关系，现在不是不显示，是显示了但显示不全，丢了一部分页面。
